test/sqllogictest/mztimestamp.slt

```diff
@@ -43,3 +43,14 @@ CREATE MATERIALIZED VIEW valid AS
 SELECT *
 FROM intervals
 WHERE mz_now() BETWEEN a AND b;
+
+statement ok
+SELECT '1702129950259'::mz_timestamp, '1990-01-04 11:00'::mz_timestamp, greatest('1990-01-04 11:00', mz_now());
```

Comment:
```suggestion
query TTT
SELECT '1702129950259'::mz_timestamp::text, '1990-01-04 11:00'::mz_timestamp::text, least('1990-01-04 11:00', mz_now())::text;
```

We should assert the results of all of these instead of just making sure they don't error.

Reply:
Done